ui/src/main/js/page/notifications/index.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+import React from 'react';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/modernmaster/forecaster/pullRequest?prid=7238741)
```suggestion
import React from "react";
```